src/program.cpp

```diff
@@ -2216,8 +2224,13 @@ bool cvk_entry_point::allocate_descriptor_sets(VkDescriptorSet* ds) {
         vkAllocateDescriptorSets(m_device, &descriptorSetAllocateInfo, ds);
 
     if (res != VK_SUCCESS) {
-        cvk_error_fn("could not allocate descriptor sets: %s",
-                     vulkan_error_string(res));
+        if (config.descriptor_sets_allocate_retry_sleep_us == UINT32_MAX) {
+            cvk_error_fn("could not allocate descriptor sets: %s",
+                         vulkan_error_string(res));
+        } else {
+            cvk_info_fn("could not allocate descriptor sets: %s",
```

Comment:
```suggestion
            cvk_warn_fn("could not allocate descriptor sets: %s",
```

Retries will have a performance impact so maybe we should flag them as warnings? Would that create too much noise in the applications you've used this with?

Reply:
Yes, it would produce too much noise. The performance impact is controlled by the performance setting the retry time. Which is disabled by default, so to me it is fine keeping it as info.
Maybe we could just warn once on first retry?